src/Easify.js

```diff
@@ -619,6 +619,24 @@
         return false;
       }
     },
+    
+    // Clones an object
+    clone: function(obj){
+      if(this.isObject(obj)) {
+        if (null == obj || "object" != typeof obj) { 
```

Comment:
You shouldn't need this `if` statement since `isObject()` already checks if it's of type "object" and already excludes it if it is `null`.